man/systemd-system.conf.xml

```diff
@@ -211,6 +204,18 @@
         kernel if the setting is greater than the actual watchdog timeout.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RuntimeWatchdogPreGovernor=</varname></term>
+
+        <listitem><para>Configure the action taken by the hardware watchdog device
+        when the pre-timeout expires. The default action for the pre-timeout event
+        depends on the kernel configuration, but it is usually to log a kernel
+        message. For a list of valid actions available for a given watchdog device,
+        check the content of the
+        <filename>/sys/class/watchdog/watchdog<replaceable>X</replaceable>/pretimeout_available_governors</filename>
+        file. If not present, the watchdog device does not support pre-timeouts.</para></listitem>
```

Comment:
maybe even list the actual name of the default governor, or of common alternative governors.